kclvm/parser/src/lib.rs

```diff
@@ -101,13 +101,40 @@ pub fn parse_expr(src: &str) -> ast::NodeRef<ast::Expr> {
     })
 }
 
+/// Build a expression from string. Note that this function is different from `parse_expr`, 
+/// it will convert some complex expressions into `StringLit` nodes.
+pub fn build_expr_from_string(value: &str) -> ast::NodeRef<ast::Expr> {
```

Comment:
Why does the function need to do some special converting for the return value of parse_expr func call? It seems a kind of tricky.

Reply:
> Why does the function need to do some special converting for the return value of parse_expr func call? It seems a kind of tricky.

Removed tricky implementation and kept it simple.

Reply:
Why is empty-check not placed inside the parse_expr method?

Reply:
> Why is empty-check not placed inside the parse_expr method?

Have moved the empty-check into the `parse_expr` method.